oauth2/oauth2-core/src/main/java/org/springframework/security/oauth2/core/endpoint/OAuth2AuthorizationRequest.java

```diff
@@ -55,6 +57,7 @@
 	private Map<String, Object> additionalParameters;
 	private String authorizationRequestUri;
 	private Map<String, Object> attributes;
+	private Instant expiresAt;
```

Comment:
An Authorization Request does not have an `expiresAt` attribute/parameter as per spec. This really is an implementation detail and should be externalized to the expiring component.